src/main/java/seedu/address/model/person/fields/Address.java

```diff
@@ -5,14 +5,13 @@
 
 import com.fasterxml.jackson.annotation.JsonValue;
 
-import seedu.address.logic.parser.exceptions.ParseException;
-
 /**
  * Represents a Person's address in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
  */
 public class Address implements Field {
 
+    public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
```

Comment:
Should this be renamed to `PREFIX`? Other classes using it can calll `Address.PREFIX` instead. Same for the other fields as well.

Reply:
This leads to a lot of naming conflicts in the tests. IntelliJ can't refactor it automatically, so it's too much refactoring for not much gain as they are basically equally readable.